src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
It's recommended to avoid using explicit width values. Consider using max-width instead to make your design more responsive.